draft-ietf-sedate-datetime-extended.md

```diff
@@ -358,14 +358,11 @@ This email header convention is in actual use, while its adaptation into
 handicapped by the fact that {{ISO8601-2000}} does not actually allow `-00:00`.
 
 Implementations that needed to express the semantics of `-00:00`
-therefore tended to use `Z` as a "neutral" offset instead.
-
-## Update to RFC 3339
+therefore tended to use `Z` instead.
```

Comment:
It might be worth, above, (line 349ish) describing the convention in 2822 and 5322 (both section 4.3, obsolete date-time forms).  That convention allows for out-of-band information to confirm the use of Z meaning +0000, while saying that otherwise Z should be interpreted as -0000.  We need, here, to overtly countermand or restate that provision for out-of-band information to make Z mean +0000.

From the de facto use of Z to mean -00:00 alluded to below, I infer that the out-of-band provision is not one known to be widely exercised, paving the way for the default reading of Z to become standard.

Reply:
The convention is described in Section 3.3 of 2822/5322.  Section 4.3 also mentions it, but that is using that convention, not defining it.